src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/Example2.java

```diff
@@ -0,0 +1,47 @@
+/*
+OperatorWrap
+option = eol
+tokens = ASSIGN, DIV_ASSIGN, PLUS_ASSIGN, MINUS_ASSIGN, STAR_ASSIGN, MOD_ASSIGN, \
+           SR_ASSIGN, BSR_ASSIGN, SL_ASSIGN, BXOR_ASSIGN, BOR_ASSIGN,BAND_ASSIGN
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
+
+class Example2 {
+  void example() {
+    int b
+            = 10; // violation ''=' should be on the previous line'
+    int c =
+            10;
+    b
+            += 10; // violation ''\+=' should be on the previous line'
+    b +=
+            10; 
+    c
+            *= 10; // violation ''*=' should be on the previous line'
+    c *=
+            10;
+    c
+            -= 5; // violation ''-=' should be on the previous line'
+    c -=
+            5;
+    c
+            /= 2; // violation ''/=' should be on the previous line'
+    c
+            %= 1; // violation ''%=' should be on the previous line'
+    c
+            >>= 1; // violation ''>>=' should be on the previous line'
+    c
+        >>>= 1; // violation ''>>>=' should be on the previous line'
+    c
+            ^= 1; // violation ''\^=' should be on the previous line'
```

Comment:
Do we have an issue created for the actual documentation example generation from the "tests"? If not, please open it and mention this nuance (escaping regexp operators). We should discuss how we can remove these in final generated documentation.

Reply:
Better to avoid regexp in Example messages.
Let's not use `^` at all, use some other or just remove it.

Reply:
Ok, after discussion in discord, we will keep necessary escapes on tokens that are in actual violation messages, and not use any regexp at all. @stoyanK7 please add to/create issue to deal with removing escapes in final generated documentation.

Reply:
https://github.com/checkstyle/checkstyle/issues/13245

Reply:
Opened at #13245. I put delimiter comments in examples.

Reply:
I would simply remove lines in examples that cause such violations.
Or remove tokens from config to avoid violation.
Example is extra big and complicated.
Better to stay compact.
It is just example, not a test, so no reason to be very detailed.

In next PR, feel freedom to cut/change example a bit to avoid such problem